www/src/_layouts/home.njk

```diff
@@ -25,6 +25,6 @@ styles:
   </div>
 </header>
 
-<main class="home__content">
+<main class="home__content" id="main-content">
```

Comment:
This also should have a `tabindex="-1"` to ensure focus is moved in all devices.